inference-engine/tests/functional/plugin/myriad/ngraph/transformations/dynamic_to_static_shape_binary_elementwise.cpp

```diff
@@ -317,37 +252,14 @@ INSTANTIATE_TEST_CASE_P(smoke_EltwiseBroadcast, DynamicToStaticShapeEltwise, tes
         ngraph::opset3::Subtract::type_info,
         ngraph::opset3::Maximum::type_info,
         ngraph::opset3::Minimum::type_info,
-        ngraph::opset3::Less::type_info),
+        ngraph::opset3::Less::type_info,
+        ngraph::opset5::Select::type_info),
     testing::Values(
         EltwiseParams{DataDims{1000}, DataDims{1}, DynamicToStaticShapeEltwise::reference_simple},
         EltwiseParams{DataDims{1000, 1, 1}, DataDims{1000, 1, 1}, DynamicToStaticShapeEltwise::reference_simple},
         EltwiseParams{DataDims{2, 1000}, DataDims{3, 1, 1}, DynamicToStaticShapeEltwise::reference_broadcast_left},
-        EltwiseParams{DataDims{1000, 64}, DataDims{1}, DynamicToStaticShapeEltwise::reference_broadcast_right})));
+        EltwiseParams{DataDims{1000, 64}, DataDims{1}, DynamicToStaticShapeEltwise::reference_broadcast_right}),
+    testing::Values(TestShapeTypes::ALL_DYNAMIC, TestShapeTypes::SINGLE_DSR)
```

Comment:
So you test `Select` with `ALL_DYNAMIC` as well? Does it mean we would have a test with `Select` where all inputs are dynamic? I didn't get if actual implementation supports that. Could you please explain?

Reply:
As I've mentioned above it doesn't matter if the first input is static or not. But yeah, it's confusing that in `ALL_DYNAMIC` mode not all of the inputs are dynamic, I've fixed it, thanks!